### PR TITLE
Updated production pem file

### PR DESCRIPTION
**Pull request by Priya5**

### CODE REVIEW

File: config/certs/apn_production.pem [hunk 1]
```diff
@@ -31,44 +31,44 @@ AQH/BAQDAgeAMBMGA1UdJQQMMAoGCCsGAQUFBwMCMBAGCiqGSIb3Y2QGAwEEAgUA
...
+LNZgPrijBi9Dao6IJ5IC2BeHvYZzacT9IRpOCvjE3ehjE9SBhkuLmVB0LcFX4U5U
+lPC6sU6mUT2o7csqBc3GV50V1w9GIXGRdiAiblhgtiFIw+7qG7GO6iobRtJLcTr2
+7wVMSqtn9UHMwC24BRXYaNS0N4ik1L/GY/XyXPyMG5PID8A/i5mAEExCv8nutbt+
+ahnIrkyy+i8zcPEB+sDemzQGSPwrIKd7/r1bLfi6+yTVhpH21xjv9zGWo7BjoM8d
+sN9ttT5Kxndp7uScI/K+S4VPeCCLK0tKha4VnU/FzNo2O7GUU9z2LCw=
 -----END CERTIFICATE-----
 Bag Attributes
-    friendlyName: FontliAPNS
-    localKeyID: 09 EB E8 9B 8D 80 B9 90 38 A4 3F F6 6B 09 2B 7C 5F 90 77 C9 
+    friendlyName: Suresh
```
Comment (padmanabhanr) [inline config/certs/apn_production.pem]: Let's retain the friendly name to `Apple Push Services: com.imaginea.fontli` rather than a individual name

Comment (Priya5) [inline config/certs/apn_production.pem]: Updated the friendly name.

**Comment by srikumarks** [comment]
Keeping production certs unencrypted in the github repo?

**Comment by Priya5** [comment]
The production cert is password protected.

**Pull request merged**